frontend/ui/time.lua

```diff
@@ -171,7 +171,7 @@ end
 function time.split_s_us(time_fts)
     if not time_fts then return nil, nil end
     local sec = math.floor(time_fts * FTS2S)
-    local usec = math.floor(time_fts - sec * S2FTS) * FTS2US
+    local usec = math.floor((time_fts - (sec * S2FTS)) * FTS2US)
```

Comment:
Good catch. The original line is buggy. Your version works. The innermost parenthesis are superfluous. So maybe write 
```
local usec = math.floor((time_fts - sec * S2FTS) * FTS2US)
```

The good news is, that this bug is not noticeable by now, as `PRECISION` can be stored in a double without precision loss; `US2FTS` is exactly 1 and `FTS2US` is exactly 1, too. But for future (if someone wants to tune the thing by using a different `PRECISION` value (e.g. a power of 2), this buggy line would cause an issue).